src/platform-apps/channels/container.tsx

```diff
@@ -68,8 +68,18 @@ export class Container extends React.Component<Properties> {
     }
   }
 
+  // only render the channel(s) which belong to "this" domain/network
+  isChannelValid(channelId) {
+    const channelIds = (this.props.channels ?? []).map((c) => c.id);
```

Comment:
> Isn't the channels list all possible channels that we've loaded so far?

no, because `this.props.channels` are being populated by `fetchChannels` saga call - [https://github.com/zer0-os/zOS/blob/main/src/store/channels-list/api.ts#L5](https://github.com/zer0-os/zOS/blob/main/src/store/channels-list/api.ts#L5), which are fetched for a particular network/domain, after which we render the chat channel.

I tested it as well, it's loading the channels for that particular domain only

Reply:
Initially i wanted to do a saga call (something like `isChannelInNetwork` at some level, assuming that's the ideal way), but this way we can avoid that extra api call..

Reply:
So, let's say we have a link in a notification and it transitions to the channels app for a different network/domain (without the page reloading) this will still be good enough to determine the channel is in the network? That means the saga must be clearing the channel list every time we load the channels app?....is it a safe assumption that this will always be the case?

Reply:
Ok, yeah, this is probably good enough for now. I think once we have proper "domains" then we can just have the domain on the channel so we can filter that way.

Reply:
> That means the saga must be clearing the channel list every time we load the channels app?....is it a safe assumption that this will always be the case?

I haven't tested with notification link. I tested by creating two different networks on my local & tried loading the channels from networkA, but actually being on networkB (in the url, eg, `/../0.networkA/.../<network-b-channel>`). The channel list was cleared & repopulated for that particular network during each load